packages/ui/cypress/e2e/case-list/filter-cases/triggersExcludedWithMultipleForces.cy.ts

```diff
@@ -0,0 +1,55 @@
+describe("Triggers excluded at force level and user level for a user with access to multiple forces", () => {
```

Comment:
A bit confused by this, looking at `allExcludedTriggers` it seems we should have a test where it only shows excluded triggers for the user if they have multiple visible forces?

Reply:
This test coverage verifies that we only exclude user-level triggers when a user has access to multiple forces. For example, when a test user has access to forces `001`, `002`, and `003`, we still render `TRPR0008` even though it's excluded for force `001` in the standing data. This is because `TRPR0008` isn't excluded at the user level.